examples/parser/Parser/Core.roc

```diff
@@ -142,6 +143,24 @@ apply = \funParser, valParser ->
 
     buildPrimitiveParser combined
 
+## Skip over a parsed item as part of a pipeline
+##
+## This is useful if you are using a pipeline of parsers with `apply` but
+## some parsed items are not part of the final result
+##
+## >>> const (\x -> \y -> \z -> Triple x y z)
+## >>> |> apply Parser.Str.nat
+## >>> |> skip (codeunit ',')
+## >>> |> apply Parser.Str.nat
+## >>> |> skip (codeunit ',')
+## >>> |> apply Parser.Str.nat
```

Comment:
Not a blocker for this PR, but personally I like the idea of changing these to `keep` and `skip` - so they're the same number of letters, and then the two alternative choices become more obivous!

Reply:
Right, that's a good suggestion.
I guess I just didn't change much that was already there in @Qqwy's original implementation unless it was necessary.
Existing usages of `apply` are in `Parser.Core` and `parse-movies-csv.roc`.
This seems to me like it should be its own PR. I'll open one now.

Reply:
#4872